meta/main.yml

```diff
@@ -11,14 +11,21 @@ galaxy_info:
   author: Reto Gantenbein
   description: 'Setup Check_MK monitoring server'
   license: 'GPL-3.0'
-  min_ansible_version: '2.1.5'
+  min_ansible_version: '2.3.0'
```

Comment:
Any reason to drop 2.2 support as well? https://github.com/debops-contrib/ansible-checkmk_server/pull/53/commits/a3a149a8e3a39becb13e33b73f701c925e05cbc6 does not mention one.